helm/reana/templates/reana-server.yaml

```diff
@@ -54,10 +54,13 @@ spec:
           - mountPath: {{ .Values.shared_storage.shared_volume_mount_path }}
             name: reana-shared-volume
           {{- range $workspace_paths := .Values.workspaces.paths }}
-          {{- if not (eq $workspace_paths "/var/reana")}}
-          - name: {{ $workspace_paths | quote |replace "/" "" }}
-            mountPath: {{ $workspace_paths }}
-            mountPropagation: HostToContainer
+          {{- if eq (typeOf $workspace_paths) "map[string]interface {}" -}}
+          {{- fail "The workspaces.paths should be a list of strings e.g cluster_node_path:shared_volume_path" -}}
```

Comment:
Minor suggestion: `{{- fail "The workspaces.paths should be a list of strings e.g node_host_path:pod_mount_path" -}}`, WDYT?